seq2science/cli.py

```diff
@@ -387,13 +388,13 @@ def _run(args, base_dir, workflows_dir, config_path):
 
     # run snakemake/seq2science
     #   1. pretty welcome message
-    setup_seq2science_logger(parsed_args)
+    setup_seq2science_logger(parsed_args, args.debug)
     log_welcome(logger, args.workflow)
 
     if args.debug:
         # dump the parsed args as readable json
         import json
-        logger.info(json.dumps(parsed_args, sort_keys=True, indent=2))
+        logger.debug(json.dumps(parsed_args, sort_keys=True, indent=2))
 
     if not args.skip_rerun or args.unlock or args.cleanup_metadata is not None:
```

Comment:
What do lines 398 to 444 still add with the newest snakemake?

Reply:
Yes I thought they could be removed as well at first. But we still somehow need to catch the `regex_patterns` since the provider changes for the assemblies.... :upside_down_face: 

Unless we somehow remove the provider param?